configs/opinicus_config

```diff
@@ -1849,6 +1849,7 @@ CONFIG_RTLWIFI_USB=m
 CONFIG_RTL8192C_COMMON=m
 CONFIG_RTL8XXXU=m
 CONFIG_RTL8XXXU_UNTESTED=y
+CONFIG_RTL8822CE=m
```

Comment:
I think you also need the  CONFIG_WLAN_VENDOR_REALTEK=y key and this was not done with menuconfig which I do recommend so you don't miss things like this...

Reply:
That was already in the file